src/services/conversionRates/getConversionRatesService.js

```diff
@@ -47,13 +97,17 @@ const _getRatesCryptocompare = async (timestamp, ratesToGet, symbol, stableCoins
   const promises = ratesToGet.map(async r => {
     const rateSymbol = stableCoins.includes(r) ? 'USD' : r;
     if (rateSymbol !== requestSymbol) {
-      const resp = JSON.parse(
-        await rp(
-          `https://min-api.cryptocompare.com/data/dayAvg?fsym=${requestSymbol}&tsym=${rateSymbol}&toTs=${timestampMS}&extraParams=giveth`,
-        ),
-      );
-
-      if (resp && resp[rateSymbol]) rates[r] = resp[rateSymbol];
+      if (requestSymbol === 'PAN') {
+        rates[r] = await _getRatesCoinGecko(rateSymbol, timestampMS, coingeckoId);
```

Comment:
It's still called inside `_getRatesCryptocompare` and not cool

Reply:
fixed